### PR TITLE
Update translations.js

### PR DESCRIPTION
**Pull request by nongpp**
add German language.

Fixes #[replace brackets with the issue number that your pull request addresses].

**Changes proposed in this pull request:**
- [list out summary of changes here]
- [list out summary of changes here]
- [list out summary of changes here]
- [etc]

*Note: pull requests without proper descriptions may simply be closed without further discussion. We appreciate your contributions, but need to know what you are offering in clearly described format. Thanks! (you can delete this text)*

cc @HospitalRun/core-maintainers

### CODE REVIEW

**Comment by nongpp** [comment]
Sorry for did not sign your Contributor License Agreement in the first time, Now I already sign it. And thank you for checked my pull request.

**Comment by tangollama** [comment]
Before we consider merging this, we need someone who actually reads and speaks German to review and approve this.

**Comment by evalsocket** [comment]
@tangollama right,translation review should be done by someone who actually reads and speaks that language

**Comment by tangollama** [comment]
@nongpp I've inquired for a German language reviewer from the community's Slack. Hopefully, someone will respond.

File: app/locales/de/translations.js [hunk 1]
```diff
@@ -18,81 +18,82 @@ export default {
     addressOptions: 'Adressoptionen',
     customForms: {
       buttons: {
-        addField: '',
-        moveDown: '',
-        moveUp: '',
-        newForm: '',
-        preview: ''
+        addField: 'Feld hinzufügen',
+        moveDown: 'Sich abwärts bewegen',
```
Comment (unused) [inline app/locales/de/translations.js]: Nach unten verschieben

File: app/locales/de/translations.js [hunk 2]
```diff
@@ -18,81 +18,82 @@ export default {
...
+        moveUp: 'Nach oben verschieben',
+        newForm: '+ neues formular',
```
Comment (unused) [inline app/locales/de/translations.js]: + Neues Formular

File: app/locales/de/translations.js [hunk 3]
```diff
@@ -18,81 +18,82 @@ export default {
...
-        property: '',
-        radio: '',
-        select: '',
-        socialworkFormType: '',
-        text: '',
-        textarea: '',
-        visitFormType: ''
+        alwaysInclude: 'Immer einschließen',
+        checkbox: 'Checkbox',
+        colSpan: 'Anzahl der Spalten zum Spannen',
```
Comment (unused) [inline app/locales/de/translations.js]: Anzahl der Spalten

File: app/locales/de/translations.js [hunk 4]
```diff
@@ -18,81 +18,82 @@ export default {
...
+        columns: 'Säulen',
```
Comment (unused) [inline app/locales/de/translations.js]: Spalten

File: app/locales/de/translations.js [hunk 5]
```diff
@@ -18,81 +18,82 @@ export default {
...
+        customForm: 'Kundenspezifisches Formular',
```
Comment (unused) [inline app/locales/de/translations.js]: eigenes Formular

**Comment by LucaDev** [comment]
Some of these translations are only half translated and still include some English words

File: app/locales/de/translations.js [hunk 6]
```diff
@@ -18,81 +18,82 @@ export default {
...
+        dischargeReportFormType: 'Entlastungsbericht',
+        epenseTo: 'Aufwand zu',
+        formName: 'Formularname',
+        formType: 'Formulartyp',
+        incidentFormType: 'Vorfall',
+        includeOtherOption: 'Include Andere Option',
+        labFormType: 'Labor',
+        label: 'Etikette',
```
Comment (unused) [inline app/locales/de/translations.js]: Bezeichnung

File: app/locales/de/translations.js [hunk 7]
```diff
@@ -18,81 +18,82 @@ export default {
...
+        newForm: 'Neue Form',
+        opdReportFormType: 'Ambulanter Bericht',
+        operativePlanFormType: 'Operativer Plan und Bericht',
+        otherOptionLabel: 'Andere Option Label',
```
Comment (unused) [inline app/locales/de/translations.js]: Bezeichnung anderer Option

File: app/locales/de/translations.js [hunk 8]
```diff
@@ -18,81 +18,82 @@ export default {
...
+        patientFormType: 'Geduldig',
```
Comment (unused) [inline app/locales/de/translations.js]: Patient

File: app/locales/de/translations.js [hunk 9]
```diff
@@ -18,81 +18,82 @@ export default {
...
+        property: 'Eigentum',
```
Comment (unused) [inline app/locales/de/translations.js]: Eigenschaft

File: app/locales/de/translations.js [hunk 10]
```diff
@@ -18,81 +18,82 @@ export default {
...
+        radio: 'Radio',
```
Comment (unused) [inline app/locales/de/translations.js]: Auswahl

File: app/locales/de/translations.js [hunk 11]
```diff
@@ -18,81 +18,82 @@ export default {
...
+        select: 'Dropdown-Liste',
+        socialworkFormType: 'Sozialarbeit',
+        text: 'Text',
+        textarea: 'Großer Text',
```
Comment (unused) [inline app/locales/de/translations.js]: Text Feld

File: app/locales/de/translations.js [hunk 12]
```diff
@@ -18,81 +18,82 @@ export default {
...
-        newCustomForm: '',
-        radioValues: ''
+        checkboxValues: 'Checkbox-Werte',
+        customForms: 'Benutzerdefinierte Formulare',
+        deleteForm: 'Benutzerdefiniertes Formular löschen',
+        dropDownValues: 'Dropdown-Werte',
+        editCustomForm: 'Benutzerdefiniertes Formular bearbeiten',
+        fields: 'Felder',
+        formSaved: 'Formular gespeichert',
+        newCustomForm: 'Neue benutzerdefinierte Formular',
```
Comment (unused) [inline app/locales/de/translations.js]: Neues benutzerdefiniertes Formular

File: app/locales/de/translations.js [hunk 13]
```diff
@@ -18,81 +18,82 @@ export default {
...
-      headerLine1: '',
-      headerLine2: '',
-      headerLine3: '',
-      logoURL: '',
-      messages: { headerSaved: '' },
-      newTitle: '',
-      titles: { optionsSaved: '' }
+      editTitle: 'Header-Optionen',
+      facilityName: 'Name der Einrichtung',
+      headerLabel: 'Header',
```
Comment (unused) [inline app/locales/de/translations.js]: Überschrift

File: app/locales/de/translations.js [hunk 14]
```diff
@@ -18,81 +18,82 @@ export default {
...
+      headerLine1: 'Header Linie 1',
```
Comment (unused) [inline app/locales/de/translations.js]: Überschrift Zeile 1

File: app/locales/de/translations.js [hunk 15]
```diff
@@ -18,81 +18,82 @@ export default {
...
+      headerLine2: 'Header Linie 2',
```
Comment (unused) [inline app/locales/de/translations.js]: Überschrift Zeile 2

File: app/locales/de/translations.js [hunk 16]
```diff
@@ -18,81 +18,82 @@ export default {
...
+      headerLine3: 'Header Linie 3',
```
Comment (unused) [inline app/locales/de/translations.js]: Überschrift Zeile 3

File: app/locales/de/translations.js [hunk 17]
```diff
@@ -18,81 +18,82 @@ export default {
...
-      editTitle: '',
-      errorDisplayAlertMessage: '',
-      errorDisplayAlertTitle: '',
-      progressMessage: '',
-      progressTitle: ''
+      displayAlertMessage: 'Bitte wählen Sie die zu ladende Datei aus.',
+      displayAlertTitle: 'Wählen Sie Datei zum Laden aus',
+      editTitle: 'Datenbank laden',
+      errorDisplayAlertMessage: 'Die Datenbank konnte nicht importiert werden. Der Fehler war: {{error}}',
+      errorDisplayAlertTitle: 'Beladung fehler',
```
Comment (unused) [inline app/locales/de/translations.js]: Fehler beim Laden

File: app/locales/de/translations.js [hunk 18]
```diff
@@ -130,77 +131,79 @@ export default {
...
-        updateConfig: '',
-        visits: ''
+        addAllergy: 'Allergie hinzufügen',
+        addAppointment: 'Termin hinzufügen',
+        addBillingDiagnosis: 'Billing-Diagnose hinzufügen',
+        addCharge: 'Gebühr hinzufügen',
+        addDiagnosis: 'Diagnose hinzufügen',
+        addImaging: 'Imaging hinzufügen',
+        addIncident: 'Vorfall hinzufügen',
+        addIncidentCategory: 'Incident Kategorie hinzufügen',
```
Comment (unused) [inline app/locales/de/translations.js]: Vorfall Kategorie hinzufügen

File: app/locales/de/translations.js [hunk 19]
```diff
@@ -130,77 +131,79 @@ export default {
...
+        addInventoryItem: 'Inventory Item hinzufügen',
```
Comment (unused) [inline app/locales/de/translations.js]: Inventar hinzufügen

File: app/locales/de/translations.js [hunk 20]
```diff
@@ -130,77 +131,79 @@ export default {
...
+        addInventoryPurchase: 'Inventory Purchase hinzufügen',
```
Comment (unused) [inline app/locales/de/translations.js]: Inventar Einkauf hinzufügen

File: app/locales/de/translations.js [hunk 21]
```diff
@@ -130,77 +131,79 @@ export default {
...
+        addInventoryRequest: 'Inventaranforderung hinzufügen',
```
Comment (unused) [inline app/locales/de/translations.js]: Inventar Anfrage hinzufügen

File: app/locales/de/translations.js [hunk 22]
```diff
@@ -130,77 +131,79 @@ export default {
...
+        fulfillMedication: 'Erfüllung Medikamente',
+        generateIncidentReport: 'Zwischenbericht erstellen',
+        imaging: 'Bildgebung',
+        incident: 'Vorfall',
+        inventory: 'Inventar',
+        invoices: 'Rechnungen',
+        labs: 'Labs',
+        listPaidInvoices: 'Liste bezahlter Rechnungen',
+        loadDb: 'Datenbank laden',
+        manageIncidents: 'Verwalten von Vorfall',
```
Comment (unused) [inline app/locales/de/translations.js]: Verwalten von Vorfällen

File: app/locales/de/translations.js [hunk 23]
```diff
@@ -130,77 +131,79 @@ export default {
...
+        medication: 'Medikation',
+        overrideInvoice: 'Override Rechnung',
```
Comment (unused) [inline app/locales/de/translations.js]: Rechnung überschreiben

**Comment by nponiros** [comment]
I had a quick look. There are a number of spelling mistakes and not everything is translated. Depending on the context in which the text is used, there might also be some wrong translations. In case nobody else can help, I might be able to commit some fixes over the weekend. Just let me know.

**Comment by jglovier** [comment]
Thanks for the feedback, everyone! ❤️ 

@nongpp could you please incorporate the feedback provided and make changes to your pull request accordingly? Thank you!

File: app/locales/de/translations.js (hunk 11 above)
Comment (caugner) [reply app/locales/de/translations.js]: *Textfeld (one word)

Comment (caugner) [reply app/locales/de/translations.js]: Or: Mehrzeiliger Text (i.e. multi-row text)

File: app/locales/de/translations.js (hunk 18 above)
Comment (caugner) [reply app/locales/de/translations.js]: Vorfall-Kategorie [with dash] (or) Kategorie für Vorfälle

File: app/locales/de/translations.js (hunk 22 above)
Comment (caugner) [reply app/locales/de/translations.js]: Vorfälle verwalten

**Comment by caugner** [comment]
What I would recommend is to split this up into two commits and have the first commit just copy the English translation to the de-folder and the second commit actually translate. Like this reviewers can have a look at the second commit and review in the context of the original.

File: app/locales/de/translations.js [hunk 24]
```diff
@@ -18,81 +18,82 @@ export default {
...
+        checkbox: 'Checkbox',
+        colSpan: 'Anzahl der Spalten zum Spannen',
+        columns: 'Säulen',
+        customForm: 'Kundenspezifisches Formular',
+        dischargeReportFormType: 'Entlastungsbericht',
+        epenseTo: 'Aufwand zu',
+        formName: 'Formularname',
+        formType: 'Formulartyp',
+        incidentFormType: 'Vorfall',
+        includeOtherOption: 'Include Andere Option',
```
Comment (mosenturm) [inline app/locales/de/translations.js]: Andere Option hinzufügen

File: app/locales/de/translations.js [hunk 25]
```diff
@@ -18,81 +18,82 @@ export default {
...
+        labFormType: 'Labor',
+        label: 'Etikette',
+        newForm: 'Neue Form',
```
Comment (mosenturm) [inline app/locales/de/translations.js]: Neues Formular

File: app/locales/de/translations.js [hunk 26]
```diff
@@ -18,81 +18,82 @@ export default {
...
-      facilityName: '',
-      headerLabel: '',
-      headerLine1: '',
-      headerLine2: '',
-      headerLine3: '',
-      logoURL: '',
-      messages: { headerSaved: '' },
-      newTitle: '',
-      titles: { optionsSaved: '' }
+      editTitle: 'Header-Optionen',
```
Comment (mosenturm) [inline app/locales/de/translations.js]: Überschrift Titel

File: app/locales/de/translations.js [hunk 27]
```diff
@@ -18,81 +18,82 @@ export default {
...
     loaddb: {
-      displayAlertMessage: '',
-      displayAlertTitle: '',
-      editTitle: '',
-      errorDisplayAlertMessage: '',
-      errorDisplayAlertTitle: '',
-      progressMessage: '',
-      progressTitle: ''
+      displayAlertMessage: 'Bitte wählen Sie die zu ladende Datei aus.',
+      displayAlertTitle: 'Wählen Sie Datei zum Laden aus',
```
Comment (mosenturm) [inline app/locales/de/translations.js]: Wählen sie eine Datei zum Laden aus

**Comment by tangollama** [comment]
@nongpp any chance you're planning to address the recommended changes?

**Pull request closed without merging**